### PR TITLE
Code Review: Issue 96: Refactor: Remove unused routes and add tests for login

### PR DESCRIPTION
**Pull request by Ghost---Shadow**
#96

### CODE REVIEW

File: test/routes/api/users/login.test.js [hunk 1]
```diff
@@ -13,5 +14,28 @@ describe('route /api/users/login', () => {
...
         .catch((e) => { throw e; });
     });
+
+    it('Should create a new user when a valid token is passed', (done) => {
+      models.users.destroy({
+        where: {
+          firstName: 'Sachi',
+        },
+      })
+        .then(() => models.facebooks.destroy({
```
Comment (mrsauravsahu) [inline test/routes/api/users/login.test.js]: You deleted the `facebooks` table entry, this is causing the `loans` GET method test to fail.

Comment (Ghost---Shadow) [reply test/routes/api/users/login.test.js]: Calling the login route should then add it back. The local tests work fine.

Comment (Ghost---Shadow) [reply test/routes/api/users/login.test.js]: Added `--runInBand` in buddy

**Comment by mrsauravsahu** [comment]
We'll do this at the end of the project. We can close this for now.

**Comment by Ghost---Shadow** [comment]
TDD requires 100% code coverage at all times. Without this the code coverage is suffering.

**Review by mrsauravsahu — APPROVED** [review]

**Pull request merged**